### PR TITLE
Custom Certs for Kibana

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-337

### CODE REVIEW

**Comment by mburke5678** [comment]
@ewolinetz PTAL

**Comment by mburke5678** [comment]
@jcantrill PTAL

File: modules/efk-logging-kibana-exposing.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+//
+// * logging/efk-logging-kibana.adoc
+
+[id='efk-logging-kibana-exposing_{context}']
+= Exposing Kibana as a route with custom certificates
+
+
+If you want to use custom certificates with Kibana, you must create a specific re-encrypt route to Kibana.  
+
+You can use the Elasticsearch secret created by the Cluster Logging Operator to generate the certificate required for the re-encrypt route.
```
Comment (jcantrill) [inline modules/efk-logging-kibana-exposing.adoc]: ... must... not can

File: modules/efk-logging-kibana-exposing.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      caCertificate: |  <3>
+          ${CA_CERT}
+      destinationCACertificate: | <4>
+          ${DEST_CA_CERT}
+      termination: reencrypt
+      insecureEdgeTerminationPolicy: Redirect
+    to:
+      kind: Service
+      name: ${SERVICE_NAME}
+parameters:
```
Comment (jcantrill) [inline modules/efk-logging-kibana-exposing.adoc]: Note in the upstreamPR that several of these params now have `required: true`

**Comment by mburke5678** [comment]
@QiaolingTang Please take a look.

**Comment by QiaolingTang** [comment]
@mburke5678 You have another [PR](https://github.com/openshift/openshift-docs/pull/14052) to remove this part, do I really need to review this PR ?

**Comment by mburke5678** [comment]
@QiaolingTang Thank you. It appears you are correct. We do not want this information in the docs. I am closing this PR. Thank you.

**Pull request merged**